tests/did/did-resolver.spec.ts

```diff
@@ -20,6 +21,20 @@ describe('DidResolver', () => {
     });
     await didResolver.resolve(did);
 
+    expect(ionDidResolveSpy.called).to.be.true;
+  });
+  it('should pick the right DID resolver based on DID method name and account for passed in cache', async () => {
```

Comment:
The test seems unrelated to the addition of a cache and does not cover the code added. The test should be along the lines of:

"should read from cache and not from resolver if cache contains the DID resolution result"